marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/EntitySearchControllerTest.java

```diff
@@ -57,30 +71,198 @@ void testCRUDOnSavedQuery() throws Exception {
         MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
         params.add("id", savedQueryResponse.get("savedQuery").get("id").asText());
 
-        getJson(SAVED_QUERIES_PATH+"/query", params)
-                .andExpect(status().isOk())
-                .andDo(result -> {
-                    ObjectNode response = readJsonObject(result.getResponse().getContentAsString());
-                    assertEquals(savedQueryResponse.get("savedQuery").get("id").asText(), response.get("savedQuery").get("id").asText());
-                });
+        getJson(SAVED_QUERIES_PATH + "/query", params)
+            .andExpect(status().isOk())
+            .andDo(result -> {
+                ObjectNode response = readJsonObject(result.getResponse().getContentAsString());
+                assertEquals(savedQueryResponse.get("savedQuery").get("id").asText(), response.get("savedQuery").get("id").asText());
+            });
 
         // testing get all savedQuery documents
         getJson(SAVED_QUERIES_PATH, new LinkedMultiValueMap<>())
-                .andExpect(status().isOk())
-                .andDo(result -> {
-                    ArrayNode response = readJsonArray(result.getResponse().getContentAsString());
-                    assertTrue(response.size() > 0, "There should be at least one saved query document");
-                });
+            .andExpect(status().isOk())
+            .andDo(result -> {
+                ArrayNode response = readJsonArray(result.getResponse().getContentAsString());
+                assertTrue(response.size() > 0, "There should be at least one saved query document");
+            });
 
         // testing delete savedQuery document
-        deleteJson(SAVED_QUERIES_PATH+"/query", params)
-                .andExpect(status().isNoContent());
-
-        getJson(SAVED_QUERIES_PATH+"/query", params)
-                .andExpect(status().isOk())
-                .andDo(result -> {
-                    ObjectNode response = readJsonObject(result.getResponse().getContentAsString());
-                    assertTrue(response.isEmpty());
-                });
+        deleteJson(SAVED_QUERIES_PATH + "/query", params)
+            .andExpect(status().isNoContent());
+
+        getJson(SAVED_QUERIES_PATH + "/query", params)
+            .andExpect(status().isOk())
+            .andDo(result -> {
+                ObjectNode response = readJsonObject(result.getResponse().getContentAsString());
+                assertTrue(response.isEmpty());
+            });
+    }
+
+
+    @Test
+    void testRowExport() throws Exception {
+        installReferenceProject(true);
+
+        Customer customer1 = new Customer(1, "Jane", 123456789, "2012-05-16");
+        Customer customer2 = new Customer(2, "John", 987654321, "2013-06-16");
+        createCustomer(customer1);
+        createCustomer(customer2);
+
+        waitForTasksToFinish();
```

Comment:
Why is this here? I don't think any post-commit triggers fire when entity instances are saved?

Reply:
Right, this needs to go. I missed that while refactoring the code.